tasklist/config/routes.rb

```diff
@@ -1,15 +1,26 @@
 Rails.application.routes.draw do
+	# Home page
 	root 'tasks#index'
-	get '/tasks/new' => 'tasks#new'
-	post '/tasks' => 'tasks#create'
 
-	get '/tasks/:id' => 'tasks#show', as: :task
-	delete '/tasks/:id' => 'tasks#delete'
+	# Add a task
+	get 'tasks/new' => 'tasks#new' # Method and the data
+	post 'tasks' => 'tasks#create' # Just the action
+
+	# Show individual page
+	get 'tasks/:id' => 'tasks#show', as: :task
+
+	# Delete
+	delete 'tasks/:id' => 'tasks#delete'
 
 	# Edit
-	# get '/update/:id' => 'tasks#edit'
+	get 'tasks/:id/edit' => 'tasks#edit'
+	patch 'tasks/:id' => 'tasks#update'
+
+	# Update to complete
+	get 'tasks/complete/:id' => 'tasks#complete'
```

Comment:
I don't think we should have both a get and a patch route for the `complete` and `incomplete` actions. Because we are actually changing the task resource when we use these routes, it makes sense to just use the patch route.